native/cocos/renderer/gfx-validator/DescriptorSetValidator.cpp

```diff
@@ -104,6 +104,11 @@ void DescriptorSetValidator::update() {
     _isDirty = false;
 }
 
+void DescriptorSetValidator::forceUpdate() {
+    _actor->forceUpdate();
+    _isDirty = false;
```

Comment:
Other backends set `_isDirty` first, should use the same logic?

Reply:
There are two types of logic.
1. DescriptorSetValidator and DescriptorSetAgent are like delegators. They just forward `forceUpdate` to its child.
2. For graphics backend like VKDescriptorSet, they will implement `forceUpdate` and call the real `update`

If we reuse base class's implementation, it will become very confusing. Since there are two possibilities.

Reply:
I agree to implement it in each backend. I just mean, should it set dirty first in `DescriptorSetValidator::forceUpdate()`.

Reply:
Since `_actor` does not known Validator, set `_isDirty = true` theoretically has not impact on `_actor`'s behaviour.
I can add some comment, but a bit reluctant to add `_isDirty = true`.

Reply:
Yep, i think just to let them have the same logic is better.

Reply:
I decided to set `_isDirty = true`.